PHP/tests/syntax_test_php.php

```diff
@@ -237,6 +237,21 @@ function foo(?stinrg ...$args) {}
 //                   ^^^ keyword.operator.spread
 //                      ^^^^^ variable.parameter
 
+$a = $b ? $c::MY_CONST : $d * 5;
+//      ^ keyword.operator.ternary
+//        ^^ variable.other
+//          ^^ punctuation.accessor.double-colon
+//            ^^^^^^^^ constant.other.class
+//                     ^ keyword.operator.ternary
+//                          ^ keyword.operator.arithmetic
+
+$a = $b ? : $c::MY_CONST;
+//      ^ keyword.operator.ternary
+//        ^ keyword.operator.ternary
+//          ^^ variable.other
+//            ^^ punctuation.accessor.double-colon
+//              ^^^^^^^^ constant.other.class
+
```

Comment:
I'm missing a test case for `:` without a `?` here. What should that be scoped as? I'd think it be a keyword still, but since the pattern for it is only pushed after a `?`, I'm not sure about it.

Reply:
I think that would be either a [label](https://github.com/sublimehq/Packages/blob/7b279e87cf54ad0ad8c0565e3dfdff5408b69f06/PHP/tests/syntax_test_php.php#L1429-L1430) or a syntax parsing error.